_posts/2021-03-09-gsoc2021.md

```diff
@@ -0,0 +1,118 @@
+
+If you are interested in working with us please read on and contact us on our [mailing list].
+[Apply] as soon as possible; the application window opens on March 29, 2021.
+
+# Project Ideas
+
+If you have your own idea about a potential project we'd love to work
+with you to develop this idea; please write to us on the [developer
+list][mailing list] to discuss it there.
+
+We also have listed several [possible projects][ideas] for you to work
+on, on our wiki. Our initial list of ideas (see summaries in the
+table below) contains 6 projects at different levels of difficulties
+and with different skill requirements. However, check the [ideas][]
+page — we might add more ideas after the posting date of this post.
+
+You don't need to have all the skills that we are listing, although
+that helps, of course. But you need to demonstrate to us that you're
+able and keen to learn anything that you don't know yet, and we will
+be happy to help you learn during your project with us.
+
+| project | name                                                                                                                                                      | difficulty | description                                                                                    | skills                      | mentors                                |
+|---------|-----------------------------------------------------------------------------------------------------------------------------------------------------------|------------|------------------------------------------------------------------------------------------------|-----------------------------|----------------------------------------|
+| 1       | [Molecular volume and surface analysis]({{ site.github.wiki }}/Project-Ideas-2021#project-1-molecular-volume-and-surface-analysis) | easy       | use an existing package for molecular surface area calculations to build a new analysis module | Python, MDAnalysis.analysis | @orbeckst, @IAlibay  |
+| 2       | [AtomGroupGroup and Rings]({{ site.github.wiki }}/Project-Ideas-2021#project-2-atomgroupgroup-and-rings)                           | medium     | Write analysis module for pi stacking analysis                                                 | Python                      | @richardjgowers, @micaela-matta, @IAlibay        |
+| 3       | [Generalise Groups]({{ site.github.wiki }}/Project-Ideas-2021#project-3-generalise-groups)                                         | medium     | Generalise concept of groups                                                                   | Python                      | @lilyminium, @fiona-naughton (?) |
+| 4       | [Cythonisation of AtomGroup]({{ site.github.wiki }}/Project-Ideas-2021#project-4-cythonise-atomgroup)                              | easy      | Cythonise AtomGroup for use with C/++                                                          | Python, Cython, C/++        | @richardjgowers, @hmacdope, @lilyminium (maybe)|
```

Comment:
If this project is thought to be too simple/easy are there other high priority areas that could be cythonised?  Just a thought.